lib/src/main/java/org/team100/lib/localization/NotePosition24ArrayListener.java

```diff
@@ -32,24 +38,53 @@ void consumeValues(NetworkTableEvent e) {
             try {
                 synchronized (m_buf) {
                     positions = m_buf.readArray(b);
+                    latestTime = Timer.getFPGATimestamp();
                 }
             } catch (RuntimeException ex) {
                 return;
             }
+
             for (NotePosition24 position : positions) {
                 // this is where you would do something useful with the payload
-                System.out.println(fields[1] + " " + position);
+                // System.out.println(fields[1] + " " + position);
             }
         } else {
             System.out.println("note weird vision update key: " + name);
         }
     }
 
-    public int getX() {
-        return positions[0].getX();
+    /**
+     * @return The x position in the camera in pixels, 0 should be the left of the
+     *         screen
+     */
+    public Optional<Double> getX() {
+        switch (Identity.instance) {
+            case BETA_BOT:
+                double xd = positions[0].getX();
+                Double x = xd;
+                Optional<Double> e = Optional.of(x);
+                return e;
+            default:
```

Comment:
i think what you're supposed to do here is use Optional.empty() as the default value.